platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -165,6 +189,7 @@ String OS_LinuxBSD::get_processor_name() const {
 			return line.split(":")[1].strip_edges();
 		}
 	}
+#endif
```

Comment:
Shouldn't the `#endif` go below?

```
ERR_FAIL_V_MSG("", String("Couldn't get the CPU model name from `/proc/cpuinfo`. Returning an empty string."));
```

Otherwise that's unreachable code for FreeBSD (since we always return before it)

Reply:
Actually, I think I should negate the `if` so that it returns a valid value and lets it fall through to this message when there is no result to return.

Reply:
Ah, I overlooked that if :eyes:, sorry.